cmd/update.go

```diff
@@ -223,6 +223,9 @@ updating process initiates.`,
 			return o.ServiceStart(ioStreams, f.QriRepoPath())
 		},
 	}
+
+	serviceStartCmd.Flags().BoolVarP(&o.Daemonize, "daemonize", "d", false, "start service as an operating system daemon")
```

Comment:
Rather than the phrase "operating system daemon", how about "long-lived daemon".